package.json

```diff
@@ -44,5 +28,9 @@
     "react-relay": "^0.9.2",
     "sync-request": "^3.0.1"
   },
-  "license" : "MIT"
+  "license" : "MIT",
+  "devDependencies": {},
+  "scripts": {
+    "install":"for sub_app in ./apps/*; do (npm install $sub_app); done"
```

Comment:
With this we can run npm install in the umbrella root and all the packages of the apps will be installed